man/git-brv.md

```diff
@@ -0,0 +1,31 @@
+git-brv(1) -- Show commit logs since some date
```

Comment:
Looks like the word is coming from `git-commits-since`? Remember to update the description here.

Reply:
Good catch ! yes indeed ;)

Reply:
fixed